spec/spec_helper.rb

```diff
@@ -52,3 +52,80 @@ def expect_push_bulk_to_match(klass, matcher)
     'args' => matcher,
   }))
 end
+
+class StreamingServerManager
```

Comment:
Can this file live somewhere other than within spec_helper?

Reply:
Yes, but I cas not sure where to put it. I can move it somewhere if you have a better idea!